src/components/ToDoList/ToDoList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { ToDoItem } from '../ToDoItem/ToDoItem';
+
+type Props = {
```

Comment:
[Not critical]
It's better to name this type something like `ToDoListProps`, so that the name describes this type the most precisely.